src/basics.md

```diff
@@ -509,8 +506,14 @@ fn open_foo() -> Result<()> {
 [`MutexGuard`]: https://doc.rust-lang.org/std/sync/struct.MutexGuard.html
 [`Mmap::as_slice`]: https://docs.rs/memmap/*/memmap/struct.Mmap.html#method.as_slice
 [`seek`]: https://doc.rust-lang.org/std/fs/struct.File.html#method.seek
-[`if let`]: https://doc.rust-lang.org/book/first-edition/if-let.html
+[`error_chain!`]: https://docs.rs/error-chain/*/error_chain/#declaring-error-types
+[`Error`]: https://docs.rs/error-chain/*/error_chain/example_generated/struct.Error.html
```

Comment:
The links with identifiers from std type linking to error_chain generated example are a nice idea in theory but will create confusion in newbie readers as well as pollute the link namespace. I would get rid of them